cypress-io/cypress
# feat: add "dot" version of "new" badge

- Closes #25739

### Additional details


This PR adds some logic do show the "dot" variant of the "new" badge in the Navigation when it is collapsed, and to change its color based on the route to make it less distracting in the spec runner page.

### Steps to test


When logged out, open a project and see that the "New" badge appears when the nav bar is expanded on the Specs List, Debug, Runs, and Settings pages. 

Collapse the nav bar and see that the badge is replaced with a green dot.

Navigate to a spec and see that the dot becomes gray when on the Spec Runner page.

Navigate back to another page and see that the dot becomes green again, and the new badge returns if the nav is expanded.

### How has the user experience changed?


https://user-images.githubusercontent.com/8340719/220725528-87f1c08b-1db2-4e35-9337-7abb59e7dacd.mp4




### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
